routes/maintenance.py

```diff
@@ -85,6 +85,16 @@ def handle(self, app, cmd):
         elif cmd == 'ssh':
             subprocess.call(['systemctl', 'restart', 'ssh'], stderr=self.void)
             return self.jsonify({'ssh': True})
+        elif cmd == 'backup':
+            subprocess.call(['tar', '-czf', '/boot/settings.tar.gz', '/root/photoframe_config'], stderr=self.void)
```

Comment:
Some thoughts here. The backup should include some meta info, preferably a JSON file with the following details:
- Last commit from running branch
- Which branch
- What version of the export tool was used (in case we need to handle different scenarios on restore)

Complete folder path should also be avoided since we cannot assume /root/ will always be the destination (in the restore).

Reply:
There's also a risk that files change during export of configuration but I think it's less of a concern